backend/src/Service/Admin/CaptainFinancialSystem/AdminCaptainFinancialSystemOneBalanceDetailService.php

```diff
@@ -18,27 +18,27 @@ public function __construct(AutoMapping $autoMapping, OrderService $orderService
         $this->autoMapping = $autoMapping;
     }
 
-    public function getBalanceDetailWithSystemOne(array $financialSystemDetail, int $captainId, float $sumPayments): AdminCaptainFinancialSystemAccordingToCountOfHoursBalanceDetailResponse
+    public function getBalanceDetailWithSystemOne(array $financialSystemDetail, int $captainId, float $sumPayments, array $date): AdminCaptainFinancialSystemAccordingToCountOfHoursBalanceDetailResponse
     {
         $countOrdersMaxFromNineteen = 0;
-        //get Count Orders
-        $countOrders = $this->orderService->getCountOrdersByCaptainId($captainId);
-        //get Orders Details
-        $detailsOrders = $this->orderService->getDetailOrdersByCaptainId($captainId);
+        //get Count Orders On Specific Date
+        $countOrders = $this->orderService->getCountOrdersByCaptainIdOnSpecificDate($captainId, $date['fromDate'], $date['toDate']);
+        //get Orders Details On Specific Date
+        $detailsOrders = $this->orderService->getDetailOrdersByCaptainIdOnSpecificDate($captainId, $date['fromDate'], $date['toDate']);
 
         foreach($detailsOrders as $detailOrder) {
            if($detailOrder['kilometer'] > CaptainFinancialSystem::KILOMETER_TO_DOUBLE_ORDER ) {
                 $countOrdersMaxFromNineteen = $countOrdersMaxFromNineteen + 1;
            }
         }
 
-        $financialSystemDetail['financialDues'] = ( ($countOrders['countOrders'] + $countOrdersMaxFromNineteen) * 
+        $financialSystemDetail['financialDues'] = ( ($countOrders['countOrder'] + $countOrdersMaxFromNineteen) * 
         
         $financialSystemDetail['compensationForEveryOrder'] ) + $financialSystemDetail['salary'];
```

Comment:
Why extended the calculation operation on two lines? Isn't better to be on the same line, and as long as you are using brackets in order to clarify and set the priorities of the operations?